lib/config.js

```diff
@@ -1,9 +1,11 @@
-/* globals process */
 var os = require('os');
 var path = require('path');
 
 module.exports = {
     DEFAULT_LOG_PATH: os.platform() !== 'win32' ? '/var/log' : path.join(process.env.ALLUSERSPROFILE, '/logs')
+    , DEFAULT_REQ_HEADERS: { 'Content-Type': 'application/json; charset=UTF-8' }
+    , DEFAULT_HTTP_ERRORS: ['ETIMEDOUT', 'ECONNRESET', 'EHOSTUNREACH', 'ETIMEDOUT', 'ESOCKETTIMEDOUT', 'ECONNREFUSED', 'ENOTFOUND']
```

Comment:
Perhaps my last comment was unclear: you've listed `'ETIMEDOUT'` twice - intentional?

Reply:
Oh, didn't notice - unintentional, of course